src/Manifest.cc

```diff
@@ -415,26 +415,38 @@ validateDepName(const StringRef name) {
     throw PoacError("dependency name must start with an alphanumeric character"
     );
   }
-  // Trailing `-`, `_`, and `/` are not allowed.
+  // Trailing `-`, `_`, `.`, and `/` are not allowed.
   if (!std::isalnum(name.back())) {
     throw PoacError("dependency name must end with an alphanumeric character");
   }
 
-  // Only alphanumeric characters, `-`, `_`, and `/` are allowed.
+  // Only alphanumeric characters, `-`, `_`, `/` and `.` are allowed
```

Comment:
Please keep the original comment style. I am using the Oxford comma.

```suggestion
  // Only alphanumeric characters, `-`, `_`, `/`, and `.` are allowed
```